src/geometry/MeshDrawMgr.cxx

```diff
@@ -131,7 +133,7 @@ void MeshDrawMgr::initVBO()
     auto drawLods = drawInfo->getDrawLods();
 
     // TODO: retrofit MeshDrawInfo to use containers
-    for (int lod = 0; lod < lodLists.size(); ++lod)
+    for (unsigned int lod = 0; lod < lodLists.size(); ++lod)
```

Comment:
sorry I missed that. A better fix would be
```c++
for (auto lod = 0u; lod < lodLists.size(); ++lod)
```
but they are functionally the same.